hack/testing/prep-host

```diff
@@ -32,3 +40,6 @@ do
 done
 
 export PATH=/usr/local/bin:$PATH
+
+sudo sed -i -e 's/--selinux-enabled//g'  /etc/sysconfig/docker
```

Comment:
Turning SELinux off can be a temporary band-aid but we need to look into why it was causing you failures. Can you reproduce the server start problem in a VM and paste here `sudo ausearch -m avc`?